nemo_asr/run_eval.py

```diff
@@ -54,15 +54,19 @@ def pack_results(results: list, buffer, transcriptions):
     return results
 
 
-def buffer_audio_and_transcribe(model: ASRModel, dataset, batch_size: int, cache_prefix: str, verbose: bool = True):
+def buffer_audio_and_transcribe(model: ASRModel, dataset, batch_size: int, pnc:bool, cache_prefix: str, verbose: bool = True):
     buffer = []
     results = []
     for sample in tqdm(dataset_iterator(dataset), desc='Evaluating: Sample id', unit='', disable=not verbose):
         buffer.append(sample)
 
         if len(buffer) == batch_size:
             filepaths = write_audio(buffer, cache_prefix)
-            transcriptions = model.transcribe(filepaths, batch_size=batch_size, verbose=False)
+
+            if pnc is not None:
```

Comment:
Minor for cleaner look: Instead of if else for each argument update PnC as true if PnC is None. And pass to func args

Reply:
Thanks for the feedback! Chose this design to support FastConformer-CTC and FastConformer-RNNT (and other future models) that don't support pnc argument in their transcribe function. This allows us to use the same buffer_audio_and_transcribe() function for all NeMo ASR models.